pkg/services/ngalert/state/compat.go

```diff
@@ -88,9 +90,14 @@ func StateToPostableAlert(transition StateTransition, appURL *url.URL) *models.P
 		return errorAlert(nL, nA, alertState, urlStr)
 	}
 
+	startsAt := strfmt.DateTime(alertState.StartsAt)
+	if featureToggles.IsEnabled(context.TODO(), featuremgmt.FlagAlertRuleUseFiredAtForStartsAt) {
```

Comment:
There's a method that doesn't require a context:
```suggestion
	if featureToggles.IsEnabledGlobally(featuremgmt.FlagAlertRuleUseFiredAtForStartsAt) {
```